cocos/core/gfx/device.ts

```diff
@@ -505,6 +505,12 @@ export abstract class GFXDevice {
      */
     public abstract createWindow (info: IGFXWindowInfo): GFXWindow;
 
+    /**
+     * @en Begin current frame.
+     * @zh 开始当前帧。
+     */
+    public abstract begin (): void;
```

Comment:
Should add end() to make API more symmetrical?

Reply:
there's already a `present` serving as end. To be more symmetrical we can use `acquire` instead of begin

Reply:
Yep, i think it is better.